vendor/README.md

```diff
@@ -202,6 +202,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **Git Config:** [Alhadis/language-etc](https://github.com/Alhadis/language-etc)
 - **Git Revision List:** [Nixinova/NovaGrammars](https://github.com/Nixinova/NovaGrammars)
 - **Gleam:** [gleam-lang/tree-sitter-gleam](https://github.com/gleam-lang/tree-sitter-gleam) 🐌
+- **Glimmer:** [atom/language-javascript](https://github.com/atom/language-javascript)
```

Comment:
🤔 this isn't right. This should be the repo path of the submodule.

I recommend running `script/add-grammar --replace` (I'm AFK so can't check the actual usage) to remove and re-add the grammar. This will call the same script as the test and it should result in the correct entry.

Reply:
This is the output of `script/add-grammar` with `--replace` or otherwise. I corrected it manually in the first place but that caused a [test failure](https://github.com/github-linguist/linguist/actions/runs/6281209531/job/17059340378):

```
  1) Failure:
TestGrammars#test_readme_file_is_in_sync [/home/runner/work/linguist/linguist/test/test_grammars.rb:42]:
Grammar list is out-of-date. Run `script/list-grammars`.
--- expected
+++ actual
@@ -194,7 +194,7 @@
 - **Git Config:** [Alhadis/language-etc](https://github.com/Alhadis/language-etc)
 - **Git Revision List:** [Nixinova/NovaGrammars](https://github.com/Nixinova/NovaGrammars)
 - **Gleam:** [gleam-lang/tree-sitter-gleam](https://github.com/gleam-lang/tree-sitter-gleam) 🐌
-- **Glimmer:** [lifeart/vsc-ember-syntax](https://github.com/lifeart/vsc-ember-syntax)
+- **Glimmer:** [atom/language-javascript](https://github.com/atom/language-javascript)
```

Maybe the `languages.yml` entry is misconfigured

Reply:
Figured it out – the `tm_scope` was incorrect in the original definition